src/test/java/data/provider/FakeRestProvider.java

```diff
@@ -0,0 +1,11 @@
+package data.provider;
+
+import com.github.javafaker.Faker;
+
+public class FakeRestProvider {
```

Comment:
Sve jasno, samo je nazovi ActivitiesProvider ako ce biti za Activities testove, ali resices to u narednom PR-u